python/tests/test_account.py

```diff
@@ -1513,7 +1488,7 @@ def test_undecipherable_group(self, acfactory, lp):
         lp.sec("ac3 reinstalls DC and generates a new key")
         ac3.stop_io()
         ac4 = acfactory.clone_online_account(ac3, pre_generated_key=False)
-        ac4.wait_configure_finish()
+        ac4._configtracker.wait_finish()
```

Comment:
This use is perhaps a bit dubious

Reply:
account objects during tests have a "_configtracker" attribute.  It could be stored outside the account in an acfactory attribute to prevent regular code in src/deltachat to access that attribute.